solidity/test/token_stake/TestTokenStake.js

```diff
@@ -801,7 +801,7 @@ contract('TokenStaking', function(accounts) {
         {from: ownerOne}
       );
 
-      await increaseTimeTo(createdAt + initializationPeriod + delegationTime + 1)
+      await increaseTimeTo(createdAt + initializationPeriod + delegationTime + timeRoundMargin)
```

Comment:
Hmm... this is weird, shouldn't margin apply to `-` cases only?

Reply:
yeah it's weird especially if you can't replicate it locally 😅 My only guess here I think if it was a normal transaction next (not .call like we have) then +1 would work reliably